app/assets/javascripts/discourse-common/addon/lib/later.js

```diff
@@ -0,0 +1,14 @@
+import { later } from "@ember/runloop";
```

Comment:
should we move debounce to common too?

Reply:
it's already in discourse-common! there is a app/lib/debounce leftover though, I hope we can remove it cleanly

Reply:
Yes that's what I meant